### DISCUSSION TITLE
fetching data via GET using Postman with Authentication Enabled in Backend Service

### DISCUSSION THREAD

**Discussion by GeraltOfRivvia** [Azure/azure-mobile-apps#588] [Q&A]
I am able to fetch data via GET using Postman when the authentication is not yet added in the Backend Service as expected. I followed the documentation provided [here](https://learn.microsoft.com/en-us/azure/developer/mobile-apps/azure-mobile-apps/quickstarts/maui/authentication?pivots=vs2022-windows). 

After adding the authentication part from the guide I wanted to test if I can get the same data with just using GET in Postman but I am getting a 401 - Unauthorized result.  I have added a bearer token generated using client credentials as below but it gives me a 401 result.

![image](https://user-images.githubusercontent.com/124562028/216920040-11928bad-a806-4750-99ec-2f365cb8f352.png)

Here is the GET for the bearer token.

![image](https://user-images.githubusercontent.com/124562028/216920541-0f2520c9-5aa3-4992-8d3f-e5a6cb09af28.png)

I also tried using the V1 of oAuth and got the same result. In the previous version of mobile app service we used for Xamarin, the same method yields a 200 or successful result with the data being retrieved.  I might just be missing something or is client credentials not applicable in generating the bearer token anymore?

**Comment by adrianhall** [comment]
A 401 indicates that the backend does not recognize the token.

Note that Azure Mobile Apps is not involved in the authentication process.  We use standard ASP.NET Core identity and authentication / authorization.  As a result, you need to make sure you are following the tutorial.  Looking at your request, you are asking for the graph.microsoft.com scope, whereas you set up an alternate scope for the Azure Mobile Apps application.  This indicates a basic misunderstanding of how Azure AD authorization works, so I would start there.
